monitoring/local_monitoring/common.sh

```diff
@@ -525,6 +529,131 @@ function kruize_local_demo_setup() {
 	fi
 }
 
+#setup the operator and deploy it
+operator_setup() {
+	echo "🔄 pulling the kruize operator repository"
+	git clone -b test-operator https://github.com/ncau/kruize-operator.git
```

Comment:
Please change this to the official repo now that all the PRs have been merged at the operator repo